ui/components/multichain/receive-modal/receive-modal.js

```diff
@@ -7,11 +7,11 @@ import {
   AvatarAccountVariant,
   Box,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import QrView from '../../ui/qr-code';
```

Comment:
Before and after showing different scrolling behaviour that will not usually get seen unless viewed at a browser height that is smaller than the extension popup

### Before 

https://github.com/MetaMask/metamask-extension/assets/8112138/d2f5f710-a91c-42ed-82bc-0d508a74cf7f

### After

https://github.com/MetaMask/metamask-extension/assets/8112138/fdcc197b-f03a-4975-8a15-6e0935e74bcd